sql/moz-fx-data-shared-prod/accounts_backend_derived/users_services_daily_v1/query.sql

```diff
@@ -88,16 +85,9 @@ windowed AS (
 SELECT
   DATE(@submission_date) AS submission_date,
   windowed.user_id_sha256,
-  oa.name AS service,
+  windowed.service,
   windowed.country,
   windowed.seen_in_tier1_country,
   windowed.registered,
-  windowed.service_events,
 FROM
   windowed
-JOIN
-  `accounts_db.fxa_oauth_clients` AS oa
-  ON windowed.service = oa.id
-WHERE
-  user_id_sha256 IS NOT NULL
-  AND service IS NOT NULL
```

Comment:
There are service names that map to multiple `id`s in `accounts_db.fxa_oauth_clients`. Because we use `service` field for joining in `users_services_last_seen`, this was resulting in a cross product there.
I see three options to fix this problem:
1. Remove the join and service name from here, add in the view
2. Add a new `service_name` column here
3. Keep the join here, but move it to an earlier stage and window over service name. This will yield less rows than current implementation and above options. The question is if here we need to distinguish between services that map to the same name in `accounts_db.fxa_oauth_clients`.

Reply:
I'm leaning towards option 1. Do you have a preference?